### PR TITLE
Backend improvement

### PR DESCRIPTION
**Pull request by PaulFenton**

### CODE REVIEW

**Comment by amine-bellamkaddem** [comment]
Paul, I think you didn't add the unit_price in ordered items. I think we need to keep track of the price of products at the moment of the order. What do you think ?

File: A2-2015/A2-Rewrite/database.sql [hunk 1]
```diff
@@ -77,7 +79,7 @@ CREATE TABLE `order_item` (
 
 /* Create companies */
 
-INSERT INTO eep_operations.company (id) VALUES ('EPP');
+INSERT INTO eep_operations.company (id) VALUES ('EEP');
```
Comment (darioalessandro) [inline A2-2015/A2-Rewrite/database.sql:82]: 👍

File: README.md [hunk 2]
```diff
@@ -71,6 +71,42 @@ If you get some weird mysql error, install mysql2 globally using:
...
+  
+Get product inventory for 'shrubs': '<url>/products?category_filter=shrubs'
+  
+Get all orders: '<url>/orders?show_pending=true&show_shipped=true'
+ 
+Get items for order #2: '<url>/order_item?order_id=2'
+  
+Create a new order: '<url>/create_order'
+  
+To create an order put the order details in the req.body:
```
Comment (darioalessandro) [inline README.md:86]: 👍 @kayvanhaverbeck

File: A2-2015/A2-Rewrite/backend/index.js [hunk 3]
```diff
@@ -209,4 +207,33 @@ app.get('/order_item', async  function (req, res) {
...
+    }).save().then(neworder => {
+         // create the associated order_item records
+        req.body.items.map(item => {
+            return OrderItems.build({
+                order_id: neworder.id,
+                product_company_id: item.product_company_id,
+                product_category: item.product_category,
+                product_code: item.product_code,
+                quantity: item.quantity,
+                unit_price: 1,
```
Comment (darioalessandro) [inline A2-2015/A2-Rewrite/backend/index.js:230]: should you do something like ```unit_price: item.price``` ?

File: A2-2015/A2-Rewrite/database.sql [hunk 4]
```diff
@@ -68,6 +67,9 @@ CREATE TABLE `order_item` (
   `product_category` varchar(30) NOT NULL,
   `product_code` varchar(10) NOT NULL,
   `quantity` float(10,2) NOT NULL,
+  `unit_price` float(10,2) NOT NULL,
```
Comment (darioalessandro) [inline A2-2015/A2-Rewrite/database.sql:70]: @amine-bellamkaddem I think this is the price.

**Review by darioalessandro — APPROVED** [review]
LGTM, PR looks good, I just found a nit when copying the price.

**Comment by amine-bellamkaddem** [comment]
@darioalessandro yes but in the backend code (/create_order) I think it is hard-coded (unit_price: 1), am I missing something ?

**Pull request merged**